src/hooks/useIsSnapInstalled/useIsSnapInstalled.tsx

```diff
@@ -116,17 +126,6 @@ export const checkIsMetaMaskImpersonator = pMemoize(
   },
 )
 
-// https://github.com/MetaMask/metamask-sdk/blob/6230d8394157f53f1b020ae44601a0a69edc6155/packages/sdk/src/Platform/PlatfformManager.ts#L102C30-L111
-export const checkisMetaMaskMobileWebView = () => {
```

Comment:
No need for this to be a function call - this got memoized in the one place it was called so effectively the same effect as importing a const - will be evaluated at the same time